### ISSUE TITLE
Help please.

### ISSUE THREAD

**Issue by shajjadul1** [naman14/Timber#400]
Can I upload it to play store. If I make changes like audio visualization, package name, theme, logo, now playing theme, etc.
Please if anybody can help me?

**Comment by anburocky3** [comment]
Yes, you can upload the app to Play Store, But make sure you follow the license provided in this repo. 
The source code must be open source and you should upload the source code in Github and provide the open source link to your users. Also, do mention the credit of the author.

These are some basic stuff you have to keep in mind before publishing.

**Comment by Lovemakery** [comment]
It doesn't run after package name is changed..please help
